packages/integrations/src/dashdot/dashdot-integration.ts

```diff
@@ -0,0 +1,156 @@
+import { humanFileSize } from "@homarr/common";
+
+import "@homarr/redis";
+
+import dayjs from "dayjs";
+
+import { logger } from "@homarr/log";
+
+import { createChannelEventHistory } from "../../../redis/src/lib/channel";
+import { Integration } from "../base/integration";
+import type { HealthMonitoring } from "../types";
+
+export class DashDotIntegration extends Integration {
+  public async testConnectionAsync(): Promise<void> {
+    const response = await fetch(this.url("/info"));
+    await response.json();
+  }
+
+  public async getSystemInfoAsync(): Promise<HealthMonitoring> {
+    const info = await this.getInfoAsync();
+    const cpuLoad = await this.getCurrentCpuLoadAsync();
+    const memoryLoad = await this.getCurrentMemoryLoadAsync();
+    const storageLoad = await this.getCurrentStorageLoadAsync();
+
+    const channel = createChannelEventHistory<CpuLoadApi[]>(`integration:${this.integration.id}:history:cpu`, 100);
+    const history = await channel.getSliceUntilTimeAsync(dayjs().subtract(15, "minutes").toDate());
+
+    return {
+      cpuUtilization: cpuLoad.sumLoad,
+      memUsed: `${memoryLoad.loadInBytes}`,
+      memAvailable: `${info.maxAvailableMemoryBytes}`,
```

Comment:
memAvailable = free memory, not total memory.
```suggestion
      memAvailable: `${info.maxAvailableMemoryBytes - memoryLoad.loadInBytes}`,
```